Helper/Cart.php

```diff
@@ -62,7 +62,7 @@
 use Bolt\Boltpay\Helper\MetricsClient;
 use Bolt\Boltpay\Helper\FeatureSwitch\Decider as DeciderHelper;
 use Magento\Catalog\Model\Config\Source\Product\Thumbnail as ThumbnailSource;
-use Zend\Serializer\Adapter\PhpSerialize as Serialize;
+use Magento\Framework\Serialize\SerializerInterface as Serialize;
```

Comment:
@danac-gs In which M2 version this class was introduced?

Reply:
2.2.0